gosamcontrib-toolfile.spec

```diff
@@ -0,0 +1,23 @@
+### RPM external gosamcontrib-toolfile 1.0
+Requires: gosamcontrib
+%prep
+
+%build
+
+%install
+
+mkdir -p %i/etc/scram.d
+cat << \EOF_TOOLFILE >%i/etc/scram.d/form.xml
```

Comment:
This is wrong, looks to be copy & paste issue. This should be `gosamcontrib.xml`.